ext/nnstreamer/tensor_filter/tensor_filter_tvm.cc

```diff
@@ -202,9 +216,9 @@ tvm_subplugin::configure_instance (const GstTensorFilterProperties *prop)
   mod_factory = tvm::runtime::Module::LoadFromFile (model_path, "so");
   gmod = mod_factory.GetFunction ("default") (device);
 
-  inputInfo.num_tensors = (int) gmod.GetFunction ("get_num_inputs") ();
-  outputInfo.num_tensors = (int) gmod.GetFunction ("get_num_outputs") ();
-
+  inputInfo.num_tensors = MIN ((int) gmod.GetFunction ("get_num_inputs") (), NNS_TENSOR_SIZE_LIMIT);
+  outputInfo.num_tensors = MIN ((int) gmod.GetFunction ("get_num_outputs") (), NNS_TENSOR_SIZE_LIMIT);
+  
```

Comment:
inputInfo.num_tensors set here again (after parsing custom prop)?
And I found trailing spaces.

Reply:
Fixed!! thx